karma.conf.js

```diff
@@ -123,6 +127,19 @@ module.exports = function(config) {
       });
     }
 
+    if (options['native-iterator']) {
+      customLaunchers = {
+        SL_Chrome: {
+          base: 'SauceLabs',
+          browserName: 'chrome'
+        },
+        SL_Firefox: {
+          base: 'SauceLabs',
+          browserName: 'firefox'
+        }
```

Comment:
The latest revision that does not have a "version" property was an attempt to use the latest version of Chrome/Firefox, because when you leave out the version it's supposed to use the latest chrome. (as explained in the link that you pointed me to)